src/components/Charts/Charts.tsx

```diff
@@ -1,4 +1,95 @@
-export default function Charts() {
-  // RENDER
-  return <div className="Charts">Charts</div>;
+// Libs
+import * as React from "react";
+import { Container } from "@material-ui/core";
+import {
+	ResponsiveContainer,
+	CartesianGrid,
+	XAxis,
+	YAxis,
+	LineChart,
+	Line,
+	Tooltip,
+	Legend,
+} from "recharts";
+
+// Assets
+import { configMonths, configGroups, iTransaction } from "../../types";
+
+export default function Charts({
+	cleanTransactions,
+}: {
+	cleanTransactions: Array<iTransaction>;
+}) {
+	// Definitions
+	const [chartData, setChartData] = React.useState<Array<{}>>([{}]);
+
+	React.useEffect(() => {
+		const groupsObj = configGroups.reduce(
```

Comment:
Utiliser reduce comme ça, c'est pas très lisible. Tu peux faire un `forEach` avec une variable passed extérieure, c'est plus facile à lire (et à écrire) je trouve.

Reply:
Replaced in later PR.